src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -269,7 +278,16 @@ void PerformInteractionRequest::on_event(const event_engine::Event& event) {
 
   if (!HasHMIResponsesToWait()) {
     LOG4CXX_DEBUG(logger_, "Send response in BOTH iteraction mode");
-    SendBothModeResponse(msg_param);
+    const bool do_send_vr_params_only =
```

Comment:
@YarikMamykin rename please `do_send_vr_params_only` -> `send_vr_params_only`

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/0190836450ab1d7e130bbb8e32315eb3325a9a68